bindings/python/capstone/x86.py

```diff
@@ -56,7 +64,7 @@ class CsX86(ctypes.Structure):
         ('addr_size', ctypes.c_uint8),
         ('modrm', ctypes.c_uint8),
         ('sib', ctypes.c_uint8),
-        ('disp', ctypes.c_int32),
+        ('disp', ctypes.c_int64),
```

Comment:
is this a mistake? there is no change on "disp" in x86.h, but it gets changed here.